src/models/GoalItem.ts

```diff
@@ -0,0 +1,9 @@
+export interface GoalItem{
+    id?: number;
+    title: string;
+    duration: string;
+    repeat: string;
+    start: Date;
+    finish: Date;
+    createdAt: Date;
```

Comment:
Do we really need this? There must be a native way in DexieJS to get an item's `createdAt` date. Please go through the API reference

Reply:
Why did you resolve this conversation? Did you find a way to do this?

Reply:
Since we haven't been able to find a way to change this to a native DexieJS value, let's go ahead and get this merged and then we can take this up shortly. @Tushar-4781 create an issue regarding the same

Reply:
Created #317 for this